ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -566,7 +561,7 @@
 								"openid-configuration"
 							]
 						},
-						"description": "The discovery endpoint can be used to retrieve metadata about the IdentityServer. It returns information like the issuer name, key material, supported scopes and endpoints."
+						"description": "Retrieves OpenID Provider configuration as described in the OpenID Connect Discovery 1.0 specification."
```

Comment:
Needs a non identity specialist description.   The OpenID provider is completely abstracted from the Team Member setting up the app so unless they have expert knowledge in OIDC they will not understand.  We can add further details links to OIDC spec but lets strive for a simpler description

Reply:
Yup, updated to:

*************

Retrieves configuration data, including:

* Endpoints.
* Default scopes. For a full list, see our documentation on <a href="https://developer-cloud.forgerock.com/reference/scopes/" target="_blank">Scopes</a>.
* Encryption and digital signature algorithms. 
* Supported claims, such as `address` and `phone_number`.